src/entity/View.ts

```diff
@@ -3,6 +3,7 @@ import { Post } from './Post';
 
 @Entity()
 @Index(['postId', 'userId'])
+@Index(['postId', 'userId', 'hidden'])
```

Comment:
Note: Once merged we should monitor carefully when it goes live. We need to have a look at the database